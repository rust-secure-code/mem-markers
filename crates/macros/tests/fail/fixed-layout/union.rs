use mem_markers::FixedLayout;

#[derive(FixedLayout)]
#[repr(C)]
union Foo {
    a: u8,
    b: u8,
}

fn main() {}
