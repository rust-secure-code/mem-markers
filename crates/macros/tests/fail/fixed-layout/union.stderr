error: FixedLayout on unions is not currently supported
 --> $DIR/union.rs:3:10
  |
3 | #[derive(FixedLayout)]
  |          ^^^^^^^^^^^
